storybook/src/data/documents.data.js

```diff
@@ -0,0 +1,60 @@
+import uuid from 'uuid/v4';
+
+export default class DocumentBuilder {
+  entityType = 'document';
+
+  uid = uuid();
+
+  systemProperties = {};
+
+  properties = {};
+
+  facets = [];
+
+  contextParameters = { favorites: { isFavorite: false }, permissions: [] };
```

Comment:
Do we need this initialised with empty permissions and a "non favorite" entry?
Should it be only `{}`?